scripts/raspbian-install.sh

```
@@ -184,49 +184,56 @@ confirm_installation() {
 
-    say "Downloading and extracting $_binary_url..."
+    say "Downloading and extracting binary from $_binary_url..."
 
     _download_dir=$(ensure mktemp -d)
 
-    # Download and extract the binary
     ensure curl -Ls "$_binary_url" | tar -xz -C "$_download_dir"
 
-    ensure sudo mkdir -p "$INSTALL_BIN_DIR"
-    ensure sudo mv "$_download_dir/wifi-connect" "$INSTALL_BIN_DIR"
+    ensure sudo mv "$_download_dir/wifi-connect" $INSTALL_BIN_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)